src/api/configuration.cpp

```diff
@@ -172,12 +179,10 @@ namespace mamba
                                 (then restart or source the contents of the shell init script))");
             }
 
-            if (!fs::exists(prefix))
-            {
-                path::touch(prefix / "conda-meta" / "history", true);
-                fs::create_directories(prefix / "envs");
-                fs::create_directories(prefix / "pkgs");
-            }
+            path::touch(prefix / "conda-meta" / "history", true);
```

Comment:
what happens if you want to initialise a root prefix in a location that already contains some files?

Reply:
ok sorry, forget what I said. this check is probably somewhere else :)

Reply:
here is just a touch and the create dir ops just fail without exception if the directories exist.
do you think it's ok ?

Reply:
yes!